package.json

```
@@ -8,7 +8,7 @@
   "scripts": {
     "lint": "standard --verbose | snazzy",
     "test": "npm run test:unit && npm run test:typescript",
-    "test:unit": "tap",
+    "test:unit": "c8 node --test test/*.js",
```

Lets add `--100` for coverage
```suggestion
    "test:unit": "c8 --100 node --test test/*.js",
```